src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/Et1VettingService.java

```diff
@@ -72,6 +111,109 @@ BEFORE_LABEL_ACAS, createDocLinkBinary(d), acasCount.incrementAndReturnValue()))
         return String.format(BEFORE_LABEL_TEMPLATE, et1Display, acasDisplay);
     }
 
+    /**
+     * Generate the Existing Jurisdiction Code list in HTML.
+     */
+    public String generateJurisdictionCodesHtml(List<JurCodesTypeItem> jurisdictionCodes) {
+        StringBuilder sb = new StringBuilder();
+        for (JurCodesTypeItem codeItem : jurisdictionCodes) {
+            String codeName = codeItem.getValue().getJuridictionCodesList();
+            if (codeName != null) {
+                try {
+                    sb.append(String.format(CASE_NAME_AND_DESCRIPTION_HTML, codeName,
+                        JurisdictionCode.valueOf(codeName.replaceAll("[^a-zA-Z]+", ""))
+                            .getDescription()));
+                } catch (IllegalArgumentException e) {
+                    log.warn("The jurisdiction code " + codeName + " is invalid.");
```

Comment:
When logging where an exception is available, always add the exception to the log too as this will help with stack traces.  I.e.
log.warn("The jurisdiction code " + codeName + " is invalid.", e);